web/app.fluidity.money/app/routes/$network/dashboard/airdrop/common.tsx

```diff
@@ -403,10 +403,11 @@ const StakingStatsModal = ({
         </LabelledValue>
         <LabelledValue label={<Text size="sm">Total Amount Staked</Text>}>
           {numberToMonetaryString(
-            getUsdFromTokenAmount(
-              stakes.reduce((sum, { amount }) => sum.add(amount), new BN(0)),
-              6
-            )
+            2 *
```

Comment:
This needs to be adjusted to accommodate for the USD value of WETH (and include comment on why this was the behaviour at first)